test/JDBC/expected/non_default_server_collation/chinese_prc_ci_as/babel_datatype.out

```diff
@@ -607,7 +607,7 @@ select CAST(2 AS money) / CAST(0.5 AS numeric(4,2));
 GO
 ~~START~~
 numeric
-4.000000
+4.0000000
```

Comment:
is this correct tsql behaviour?

Reply:
This difference is due because, currently money datatype typmod is considered as -1, but should have been considered as corresponding to precision = 19, scale = 4. TSQL returns result till 6 scale. This difference should get fixed once the above issue with money & smallmoney is fixed.